core.md

```diff
@@ -57,23 +57,13 @@ You may [apply to join some teams](https://links.neoforged.net/apply), but some
 
 #### Steering Council
 
-The Steering Council is the overall management and oversight of the Project.
+The Steering Council helps to ensure good communication within teams by making sure significant work, PRs, rule changes, or decision are backed by a team's vote that is outlined in [/voting.md](/voting.md). The result of that team's vote will be then enforced, whether it be the Maintainer team, Moderator team, or any other. While any team member can put up votes themselves, the Steering Council acts as a dedicated monitoring position to watch for anything that could slip by without a vote.
 
-They “steer” the project to keep it aligned with the long-term vision, hence its secondary name of Vision Team.
+In other words, the members of the Steering Council does not give out orders or demands. They cannot tell people what to do. Instead, if something comes up and a majority of the Steering Council members believes it is significant, they can say that the thing needs a decision from the whole team and put up a vote for that team. This helps teams be aware of what others members on their team are doing and helps reduce people being blindsided by big changes. It also addresses the problem of a single member making sweeping changes under other member's noses.
```

Comment:
The wording of the first part of this paragraph should be improved a little: "something", "thing" should be clarified.

Regarding the second part: the "blindsided" and "single member making sweeping changes" parts do not belong here, and I would just remove them.

Reply:
Oh you must had been reviewing before I just pushed up the new wording. The something and thing was replaced moment ago